src/Digdir.Domain.Dialogporten.Application/LocalDevelopmentSettings.cs

```diff
@@ -15,6 +15,7 @@ public sealed class LocalDevelopmentSettings
     public bool UseLocalDevelopmentOrganizationRegister { get; set; } = true;
     public bool UseLocalDevelopmentCompactJwsGenerator { get; set; } = true;
     public bool UseInMemoryServiceBusTransport { get; set; } = true;
+    public bool DisablePolicyInformationSyncOnStartup { get; set; } = true;
```

Comment:
Burde vi hatt en slik en for DevelopmentSubjectResourceSyncHostedService (DisableSubjectResourceSyncOnStartup)?

Reply:
Jepp, gir mening det.